documentation/book/assembly-resource-limits-and-requests.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-resource-limits-and-requests-{context}']
+
+= CPU and memory resources
+
+For every deployed container, {ProductName} allows you to specify the resources which should be reserved for it and the maximal resources which should be consumed by it.
```

Comment:
For every deployed container, {ProductName} allows you to specify the resources which should be reserved for it and the maximal resources which should be consumed by it.

TO

For every deployed container, {ProductName} allows you to specify the resources which should be reserved for it and the maximum resources that can be consumed by it.